zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -1,21 +1,16 @@
 import zbsutils
-import zstacklib.utils.jsonobject as jsonobject
-
-from zstacklib.utils import plugin
 from zstacklib.utils import daemon
-from zstacklib.utils import linux
-from zstacklib.utils import traceable_shell
 from zstacklib.utils import iproute
-from zstacklib.utils.report import *
+from zstacklib.utils import plugin
+from zstacklib.utils import traceable_shell
 from zstacklib.utils.bash import *
```

Ruff (0.12.2)

`from zstacklib.utils.bash import *` used; unable to detect undefined names

(F403)